### PR TITLE
Added keywords

### PR DESCRIPTION
**Pull request by astrofrog**
This will help people find astropy when searching for these terms in PyPI.

Any other suggestions?

### CODE REVIEW

**Comment by pllim** [comment]
What about the other sub-packages we support, like WCS, VO, SAMP, etc?

**Comment by hamogu** [comment]
ASCII reader, table, coordinate conversion, fits, physical units

**Comment by mwcraig** [comment]
Restarted doc ubild fixed by #4547

**Comment by astrofrog** [comment]
I've added many of the suggestions - any other ones for now?

**Comment by astrofrog** [comment]
I'm going to go ahead and merge this - @eteq, I think this should be backported for the releases to make sure we add the keywords to PyPI for the new releases - do you agree?

File: setup.py [hunk 1]
```diff
@@ -93,6 +93,9 @@
       url='http://astropy.org',
       long_description=astropy.__doc__,
       download_url='%s/astropy-%s.tar.gz' % (DOWNLOAD_BASE_URL, VERSION),
+      keywords=['astronomy', 'astrophysics', 'cosmology', 'space', 'science',
+                'units', 'table', 'wcs', 'vo', 'samp', 'table', 'coordinate',
+                'fits', 'modeling', 'models', 'fitting', 'ascii'],
```
Comment (saimn) [inline setup.py]: `table` is duplicated ;)

**Comment by eteq** [comment]
Agree with backporting for the bugfix releases to gain the keywords.  Will merge now (manually with the fix @saimn noted)

**Pull request merged**